migrations/db_scripts

```diff
@@ -0,0 +1,27 @@
+create database chitter
```

Comment:
Extracting these scripts out feels like a good step in the direction of automating db setup.  If you want to continue down this path take a look at what you can do with a Rakefile